src/Factory/Api/Mastodon/Application.php

```diff
@@ -22,28 +22,41 @@
 namespace Friendica\Factory\Api\Mastodon;
 
 use Friendica\BaseFactory;
-use Friendica\Database\DBA;
+use Friendica\Database\Database;
+use Friendica\Network\HTTPException\InternalServerErrorException;
+use Psr\Log\LoggerInterface;
 
 class Application extends BaseFactory
 {
+	/** @var Database */
+	private $dba;
+
+	public function __construct(LoggerInterface $logger, Database $dba)
+	{
+		parent::__construct($logger);
+		$this->dba = $dba;
+	}
+
 	/**
 	 * @param int $id Application ID
+	 *
+	 * @return \Friendica\Object\Api\Mastodon\Application
+	 *
+	 * @throws InternalServerErrorException
 	 */
 	public function createFromApplicationId(int $id)
```

Comment:
Please add the return type hint.
```suggestion
	public function createFromApplicationId(int $id): \Friendica\Object\Api\Mastodon\Application
```